yarn.lock

```diff
@@ -2695,6 +2695,13 @@
     uuid "^8.0.0"
     xdg-basedir "^4.0.0"
 
+"@growthbook/growthbook-react@^0.12.0":
+  version "0.12.0"
+  resolved "https://registry.yarnpkg.com/@growthbook/growthbook-react/-/growthbook-react-0.12.0.tgz#c97d40d3cc22b56d04e170d9d7149ea1567fac6f"
+  integrity sha512-Oo2cbGpM8thaAmwEccKFy/uRU3bqU4r4xcpKaLEHPeGARCkccFi1tdq2GPYEoO6cyeo7/Wc9eHqhtazg340orA==
+  dependencies:
+    "@growthbook/growthbook" "^0.23.0"
```

Comment:
These changes seem unrelated to your documentation-related changes and are likely to create conflicts with other PR's that change dependencies.

Reply:
oops